tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -67,7 +63,7 @@ def post_status_for_file_to_queue(
     request_method: RequestMethod,
     db_queue_url: str,
 ):
-    new_data = {"job_id": job_id, "granule_id": granule_id, "filename": filename}
+    new_data = {"job_id": job_id, "granule_id": granule_id, "filename": filename, "last_update": last_update}
```

Comment:
The following items should also be a part of your "new_data" initialization:
- status_id

Depending on where we want the business logic to live, we should also do checks to set completion_time if status_id is in a Success or failure state and validate that error_message is set if status_id is failure. This is something we should quick discuss as a team with @AndrewDorn